internal/resources/metal/project/sweeper.go

```diff
@@ -0,0 +1,44 @@
+package project
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/equinix/terraform-provider-equinix/internal/sweep"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func AddTestSweeper() {
```

Comment:
This pattern is consistent with https://github.com/hashicorp/terraform-provider-aws/blob/e8e0ab740dae669a06b82f0a5a27ca7d1f311df3/internal/service/elasticache/sweep.go#L27 

Aside from having a different function name:
- sweeper is exposed as a package exported function
- calls `resource.AddtestSweepers` directly
- sweeper resides in non `_test.go` file